pkg/testcase/selinux.go

```diff
@@ -0,0 +1,106 @@
+package testcase
+
+import (
+    "fmt"
+	"github.com/rancher/distros-test-framework/pkg/assert"
+	"github.com/rancher/distros-test-framework/factory"
+	"github.com/rancher/distros-test-framework/shared"
+
+	. "github.com/onsi/ginkgo/v2"
+)
+
+
+// Validate that containerd is running with selinux enabled in the config
+func TestSelinuxEnabled(product string) {
+	ips := shared.FetchNodeExternalIP()
+	selinuxConfigAssert := "selinux: true"
+	selinuxContainerdAssert := "enable_selinux = true"
+
+	for _, ip := range ips {
+		assert.CheckComponentCmdNode("cat /etc/rancher/"+product+"/config.yaml", ip, selinuxConfigAssert)
+		assert.CheckComponentCmdNode("sudo cat /var/lib/rancher/"+product+"/agent/etc/containerd/config.toml", ip, selinuxContainerdAssert)
+	}
+}
+
+// Validate container-selinux version, rke2-selinux version and rke2-selinux version
+func TestSelinuxVersions(product string) {
+	cluster := factory.GetCluster(GinkgoT())
+
+	var serverAsserts [] string
```

Comment:
you can join this var declaration with the above since they are doing the same "thing" 


```
cluster := factory.GetCluster(GinkgoT())
var serverAsserts []string

```